packages/daemon/test/test-endo.js

```diff
@@ -41,6 +42,36 @@ const makeLocator = (...root) => {
   };
 };
 
+// The id of the next bundle to make.
+let bundleId = 0;
+const textEncoder = new TextEncoder();
+
+// TODO: We should be able to use {import('../src/types').EndoHost} for `host`,
+// but when laundered through `E()` it becomes `never`.
```

Comment:
@kriskowal @kumavis is this something you've encountered before? Is it user error?

Reply:
No. The `.js` extension may be required (even though it is a `.d.ts`)

Reply:
This does not appear to make a difference on my system. The error is visible in my editor (VSCode) using the repo version of TypeScript (`5.3.3`). ESLint does not appear to be configured to validate JSDoc types (although we could make it so).